engine/src/main/java/com/redhat/cloud/notifications/db/repositories/DrawerNotificationRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.redhat.cloud.notifications.db.repositories;
+
+import com.redhat.cloud.notifications.db.StatelessSessionFactory;
+import com.redhat.cloud.notifications.models.DrawerNotification;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.List;
+
+@ApplicationScoped
+public class DrawerNotificationRepository {
+    @Inject
+    StatelessSessionFactory statelessSessionFactory;
+
+    public DrawerNotification create(DrawerNotification drawerNotification) {
+        drawerNotification.prePersist(); // This method must be called manually while using a StatelessSession.
+        statelessSessionFactory.getCurrentSession().insert(drawerNotification);
+        return drawerNotification;
+    }
+
+    public List<DrawerNotification> getDrawerNotificationsByUserId(String userId) {
+        String query = "SELECT dn FROM DrawerNotification dn WHERE userId = :userId";
+        return statelessSessionFactory.getCurrentSession().createQuery(query, DrawerNotification.class)
+            .setParameter("userId", userId)
+            .getResultList();
+    }
```

Comment:
The drawer notifications will be retrieved from the `backend` REST API which means we'll use a different repository in the `backend` module for that. You can move this method to the test class to keep testing the DB records creation.
```suggestion
```

Reply:
Note for later: we'll always include the `orgId` in the queries to retrieve drawer notifications because two orgs may contain the same `userId`.

Reply:
Ho I see, I wasn't aware that was possible !
Do you know how a user can longin is such case ? Because on console dot login page the org id is not required ?